drivers/crypto/crypto_stm32.c

```diff
@@ -159,24 +188,23 @@ static int crypto_stm32_ecb_decrypt(struct cipher_ctx *ctx,
 		return -EINVAL;
 	}
 
-	ret = do_decrypt(ctx, pkt->in_buf, pkt->in_len, pkt->out_buf);
+	ret = do_aes(ctx, hal_ecb_decrypt_op, pkt->in_buf, pkt->in_len, pkt->out_buf);
 	if (ret == 0) {
 		pkt->out_len = 16;
 	}
 
 	return ret;
 }
 
-static int crypto_stm32_cbc_encrypt(struct cipher_ctx *ctx,
-				    struct cipher_pkt *pkt, uint8_t *iv)
+static int crypto_stm32_cbc_encrypt(struct cipher_ctx *ctx, struct cipher_pkt *pkt, uint8_t *iv)
```

Comment:
Avoid style changes if not required, it confuses review w/o additional review.
If really required, make it a dedicated commit.